src/ontology/fypo-odk.yaml

```diff
@@ -15,17 +15,33 @@ release_artefacts:
   - full
 primary_release: full
 import_group:
+  use_base_merging: TRUE
+  exclude_iri_patterns:
+    - <http://purl.obolibrary.org/obo/UBERON_*>
   products:
+    - id: go
+      make_base: TRUE
+      base_iris:
+        - http://purl.obolibrary.org/obo/GOCHE_
+        - http://purl.obolibrary.org/obo/GO_
+        - http://purl.obolibrary.org/obo/GOREL_
     - id: ro
+      use_base: TRUE
     - id: iao
+      use_base: TRUE
     - id: chebi
+      use_base: TRUE
       mirror_from: https://raw.githubusercontent.com/obophenotype/chebi_obo_slim/main/chebi_slim.owl
-    - id: go
     - id: cl
+      use_base: TRUE
     - id: pato
+      use_base: TRUE
     - id: so
+      use_base: TRUE
     - id: bto
+      use_base: TRUE
     - id: pr
+      use_base: TRUE
```

Comment:
Same as for CHEBI above. `use_base` is not really needed here (though it’s not harmful, it’s just ignored).